Makefile.am

```diff
@@ -1,9 +1,12 @@
+SUBDIRS = \
+	package_modules \
+	tests/unit
```

Comment:
Why is tests/unit/ in SUBDIRS but tests/acceptance is in TESTS_SUBDIRS ?
Surely the logic should be:
```
SUBDIRS = package_modules
TESTS_SUBDIRS = tests/unit
if HAVE_CORE
    TESTS_SUBDIRS += tests/acceptance
endif
SUBDIRS += $(TESTS_SUBDIRS)
```